src/test/common/steps/guidance-steps.js

```diff
@@ -0,0 +1,46 @@
+const { When, Then } = require('cucumber')
+const { expect } = require('chai')
+
+const pages = require('./pages')
+
+When(/^I open the (.*) guidance page$/, async function (pageName) {
+  await pages.guidance.openGuidancePage(pages.url, pageName)
+})
+
+Then(/^all the (.*) guidance page content is present$/, async function (pageName) {
+  await assertGuidancePageHeadersCorrect(pageName)
+  await assertGuidancePageContentsCorrect(pageName)
+  await assertGuidancePageNavigationLinksCorrect(pageName)
+})
+
+async function assertGuidancePageHeadersCorrect (pageName) {
+  const h1Text = await pages.cookies.getH1Text()
+  expect(h1Text.toString().trim()).to.be.equal('Get money off milk, food and vitamins (Healthy Start)',
+    'expected guidance page H1 text to be correct')
+  const h2Text = await pages.cookies.getH2Text()
+  let expectedH2PageName = pageName
+  if (pageName === 'What you get') {
+    // Page title from pages doesn't match what's in the title of the What you get page
```

Comment:
Does this mean the `PAGES` object is incorrect for “What you get”? Can we just update `PAGES`?

Reply:
No, I tried this but tests fail sadly, I'm not sure whether it's worth trying to fix those tests or leave this like this, what do you think?

Reply:
I would try and fix as I don't think its great to have this sort of check. Maybe the page objects in `PAGES` need updating to:
```
{
  title: 'What you’ll get',
  path: '/what-you-get',
  label: 'What you get'
}
```

Reply:
Ok, it turned out to be easier than I thought to fix this, its sorted now and this extra check has been removed.